kong/api/routes/health.lua

```diff
@@ -64,6 +67,15 @@ return {
         },
       }
 
+      -- if dbless mode is enabled we provide the current hash of the
+      -- data-plane in the status response as this enables control planes
+      -- to make decisions when something changes in the data-plane (e.g.
+      -- if the gateway gets unexpectedly restarted and its configuration
+      -- has been reset to empty).
+      if dbless or data_plane_role then
+        status_response.configuration_hash = declarative.get_current_hash()
```

Comment:
Can we make this simply `null` in case there is no config yet? I think `"no_configuration_yet"` magic string is a bit more confusing.

Also, in LMDB we are currently changing the "empty config hash" to `string.rep("0", 32)`. Therefore, maybe you can store the same data `string.rep("0", 32)` inside the shdict in this PR and in here, export `null` in case the hash is `string.rep("0", 32)`.